### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by kenymc1**

### CODE REVIEW

**Comment by davideverett89** [comment]
👍

File: Fan.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    />
+    <script
+      src="https://kit.fontawesome.com/1c3c2830bd.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://fonts.googleapis.com/css?family=Alfa+Slab+One|Lato&display=swap"
+      rel="stylesheet"
+    />
+    <title>Home</title>
```
Comment (ancasimon) [inline Fan.html:21]: All the html pages have Home as the title in the head

File: main.js [hunk 2]
```diff
@@ -1 +1,47 @@
-console.log("Battleheart");
\ No newline at end of file
+const tourDates = [
+  {
+    date: 'Fri May 1',
+    venue: 'Lambo Festival 2020'
```
Comment (ancasimon) [inline main.js:4]: Need a comma in the tour dates’ array after the venue key

**Comment by ancasimon** [comment]
Looks good - we can fix those two things afterwards too

**Pull request merged**